.changeset/strong-phones-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/cache.gold-adapter': patch
```

Comment:
Exports change is backwards incompatible, right?

```suggestion
'@chainlink/cache.gold-adapter': minor
```

Reply:
oh right, should be minor